Externals/hidapi/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(hidapi)
+
+add_library(hidapi hidapi/hidapi.h)
+target_include_directories(hidapi PUBLIC hidapi)
+
+if(APPLE)
+  target_sources(hidapi PRIVATE mac/hid.c)
+elseif(MSVC)
+  target_sources(hidapi PRIVATE windows/hid.c)
+else()
+  find_package(Libudev)
+  if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux" AND LIBUDEV_FOUND)
```

Comment:
You need to quote this or remove the ${} as it may do double expansion. If there was a "Linux" variable, that's what you'd be checking!